src/main/core/Entity/Log/LogSecurity.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Claroline\CoreBundle\Entity\Log;
+
+use Claroline\CoreBundle\Entity\User;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="claro_log_security")
+ */
+class LogSecurity
```

Comment:
`SecurityLog` sounds better, I suggest to rename for consistency.
About the table name, maybe `claro_logs_security`? I don't remember our naming policy (plural or singular?)

Reply:
I changed the filename and the naming policy is singular, thx :)